### ISSUE TITLE
Midterm review by bje43

### ISSUE THREAD

**Issue by Benedds** [annasxu/speed-dating#8]
This paper is analyzing data from a number of speed dating events in order to give insight into what factors determine whether you liked someone on a date and how well people are able to assess their own dates. This report focused mostly on analyzing the correlation between variables such as race, gender, their own ratings of themselves, and their date's rating of them using linear models.

Things I liked: 
	1) Defining you're own self awareness score was a really good idea and the metric you picked was very intuitive to understand. 
	2) While reading the start of the section on self-awareness I was thinking about how interesting it would be to look at the difference between how people saw themselves and how people thought other saw themselves and you addressed this immediately after and summarized it very nicely. 
	3) All the figures were easy to analyze and had very descriptive captions but were not repetitive with the analysis done in the text.

Things I think can be improved:
	1) I would have liked for you to use other models besides linear regression for the analysis. I also wish you gave a little bit more detail into how the linear regression models were formed and what their coefficients were.
	2) I would have liked for the next steps section to be have more details. Some questions I have after reading that section are "What methods will you use to produce linear models?" and "How are you going to divide your test and training set"
	3) You don't address how you plan to avoid overfitting or underfitting as the writeup suggested.
